src/bench.h

```diff
@@ -56,21 +56,29 @@ void print_number(const int64_t x) {
     /* Format and print the number. */
     ptr = sizeof(buffer) - 1;
     buffer[ptr] = 0;
-    if (c != 0) {
+    g = 0;
+    if (c != 0) { /* non zero fractional part */
         for (i = 0; i < c; ++i) {
             buffer[--ptr] = '0' + (y % 10);
             y /= 10;
         }
-        buffer[--ptr] = '.';
+    } else if (c == 0) { /* fractional part is 0 */
+        buffer[--ptr] = '0'; 
     }
+    buffer[--ptr] = '.';
     do {
         buffer[--ptr] = '0' + (y % 10);
         y /= 10;
+        g++;
     } while (y != 0);
     if (x < 0) {
         buffer[--ptr] = '-';
+        g++;
     }
-    printf("%s", &buffer[ptr]);
+    printf(", "); /* for center alignment */
```

Comment:
```suggestion
    printf(", "); /* as a column delimiter */
```
I think that's easier to grasp if you haven't read the discussion in this PR.